examples/DESC/SOLOVEV

```diff
@@ -3,41 +3,36 @@
 # global parameters
 sym = 1
 NFP = 1
-Psi = 1.00000000E+00
+Psi = 1.0
 
 # spectral resolution
 M_pol  =  6, 12
 N_tor  =  0
 M_grid = 12, 18
 N_grid =  0
 
-# continuation parameters
-bdry_ratio = 1
-pres_ratio = 0, 1
```

Comment:
@ddudt any reason for getting rid of the pressure perturbations here?

Reply:
We run the SOLOVEV example with the --numpy flag in our tests, and perturbations without JAX means finite differences which are inaccurate right? So I imagine that is the reason for dropping the perturbation here

Reply:
I think that's why I had them in there, to make sure we tested that code path. It's a simple enough case that the finite differences still have reasonable results (at least compared to vmec)

Reply:
though actually, it looks like its not running the numpy backend even with the flag set, I think because it already imports the jax backend. We might need to run that test in a subprocess or something.

On a related note, I think it might make sense to have a separate set of input files that we use for testing, apart from the example folder. So the examples can really be examples of good parameters, and the test ones can be lower res / exercise more of the code. thoughts @dpanici @ddudt ?

Reply:
I think that is a good idea, I feel like at one point it was like this. I can make this change tomorrow